packages/components/src/internal/app/utils.test.ts

```diff
@@ -823,10 +828,51 @@ describe('utils', () => {
     });
 
     test('isCommunityDistribution', () => {
-        expect(isCommunityDistribution({})).toBeTruthy();
-        expect(isCommunityDistribution({ api: { moduleNames: ['samplemanagement'] } })).toBeFalsy();
-        expect(isCommunityDistribution({ api: { moduleNames: ['premium'] } })).toBeFalsy();
-        expect(isCommunityDistribution({ api: { moduleNames: ['api'] } })).toBeTruthy();
+        expect(isCommunityDistribution({ api: { moduleNames: ['api'] } })).toBeTruthy(); // LKS Community
+        expect(isCommunityDistribution({ api: { moduleNames: ['samplemanagement'] } })).toBeFalsy(); // LKSM-only
+        expect(isCommunityDistribution({ api: { moduleNames: ['premium'] } })).toBeFalsy(); // LKS Starter
```

Comment:
seems like `samplemanagement` should be included in the `modulenames` for the distributions that have it so we assure the mere presence of that module doesn't change the answer.

Reply:
good call. added.